public/js/mage/adminhtml/tools.js

```diff
@@ -754,3 +755,74 @@ function copyText(event) {
         copyIcon.classList.remove('icon-copy-copied');
     }, 1000);
 }
+
+/**
+ * Clear <div id="messages"></div>
+ */
+function clearMessageDiv() {
+    setMessageDivHtml('');
+}
+
+/**
+ * Set a message in <div id="messages"></div>
+ *
+ * @param {string} message - text value of the message to display
+ * @param {string} type - one of `success|error|notice`
+ */
+function setMessageDiv(message, type = 'success') {
```

Comment:
```suggestion
function setMessagesDiv(message, type = 'success') {
```

since the target div is #messages?